app/api/v1/defaults.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../defaults'
+
+module API
+  module V1
+    module Defaults
+      def self.included(target)
+        target.send :include, API::Defaults
```

Comment:
[NITPICK] `Module#include` is public method (since 2.2, I think).

```
$ ruby -ve 'class Foo; end; Foo.include(Module.new)'
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
```